Exercise_1/Exercise_1.ipynb

```diff
@@ -18,8 +18,40 @@
     "   * **init_exercise:** It will create the *work* directory and navigate into it \n",
     "   * **reset:** It will delete the *work* folder and enable you a clean restart of the exercise in case you completely mess it up.\n",
     "   \n",
-    "In the following (and all further exercises), all text enclosed with `<>` denotes a placeholder to be replaced by a specific string appropriate in **your** context.\n",
-    "   "
+    "In the following (and all further exercises), all text enclosed with `<>` denotes a placeholder to be replaced by a specific string appropriate in **your** context.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "considered-session",
+   "metadata": {},
+   "source": [
+    "### Start exercises in correct folder\n",
+    "This exercise (and all further exercises) assume that the shell is already located in the folder of the exercise notebooks. For some reason, depending on your machine, the notebook does not switch to the folder of the notebook.\n",
```

Comment:
I would change the text slightly: 
It could happen, for some reason, that the notebook does not by default switch to the folder of the notebook.